lib/mix/tasks/ecto.drop.ex

```diff
@@ -28,6 +28,18 @@ defmodule Mix.Tasks.Ecto.Drop do
 
     {opts, _, _} = OptionParser.parse args, switches: [quiet: :boolean]
 
+    if disable_safety_warnings?(repo) or Mix.shell.yes?("Are you sure you want to drop the database for repo #{inspect repo}?") do
```

Comment:
If we compare with true (Atom)
Only 
disable_safety_warnings: true
If destructive without question mark?